cmake/templates/win32_python/__init__.py

```diff
@@ -0,0 +1,20 @@
+def _fixup_dlls():
+    try:
+        path = __path__[0]
+    except (NameError, IndexError):
+        return # Not a package, or not on filesystem
+
+    import os
+
+    relpath = os.path.relpath(path, __path__[-1])
+    dll_path = os.path.abspath(os.path.join(__path__[-1], '../bin', relpath))
+    if not os.path.isdir(dll_path):
+        return
+
+    os_path = os.environ.get('PATH', '')
+    os_path = os_path.split(os.pathsep) if os_path else []
+    os_path.insert(0, dll_path)
+    os.environ['PATH'] = os.pathsep.join(os_path)
```

Comment:
The function here is to ensure that `bin` is in the search path for DLLs. Does Python 3 somehow prevent the DLL loader from seeing changes to the environment? At what version should I begin to use `os.add_dll_directory` instead of touching PATH?

Reply:
That would be Python 3.8 (you can simply check whether that function exists).  See the `panda3d/__init__.py` file that makepanda generates on Windows (I posted an excerpt of the generation logic in another comment here), which has proven to work so I see no reason to replace it with new logic.